src/entries/popup/pages/home/Points/PointsDashboard.tsx

```diff
@@ -892,6 +892,11 @@ function MyEarnings({ earnings = '0' }: { earnings?: string }) {
     decimals: 18,
     symbol: eth?.nativeAsset?.symbol,
   });
+  const earningsWithThreshold = handleSignificantDecimalsWithThreshold(
+    convertRawAmountToDecimalFormat(earnings, 18),
+    6,
+    '0.000001',
```

Comment:
why with threshold and not just `handleSignificantDecimals`?

Reply:
I saw them both and just found this easier to understand at first so i used it haha. Is there a significant difference between the two where i shouldn't use withThreshold?

Reply:
is just one additional param that i don't think we need but won't block